scripts/make_port_status.py

```diff
@@ -113,9 +113,9 @@ def get_mathlib4_module_commit_info(contents):
         capture_output=True)
     pr_matches = re.search(r'(#[0-9]+)\)$', log.stdout.decode().splitlines()[-1])
     if pr_matches:
-        mathlib4_pr = 'mathlib4' + pr_matches.groups()[0]
+        mathlib4_pr = int(pr_matches.groups()[0])
```

Comment:
The match starts with `#`. So this will error.

Reply:
Nice catch, fixed